### DISCUSSION TITLE
General help with setting up?

### DISCUSSION THREAD

**Discussion by darkshoxx** [lifeiteng/valle#161] [Q&A]
Hi,

I'm very new to all of this. 
I've cloned the repo and made a docker image. (Win 11 WSL Docker Desktop)
I can run the image as containers, but don't know what to do with them after that.
Assuming I have an audio-input file "input.wav" (together with its transcript "transcript.txt" and a text prompt "prompt.txt"
How do I proceed? Or how do I run the examples?
I just don't know what the next steps are, can someone give me any pointers?

**Comment by guoaoo** [comment]
You could utilize the LJSpeech dataset to explore data processing techniques. Please visit the following URL: https://github.com/lifeiteng/vall-e/tree/main/egs/ljspeech#ljspeech for more details. It is important to read the contents of the file called [prepare.sh](https://github.com/lifeiteng/vall-e/blob/main/egs/ljspeech/prepare.sh).

**Comment by lifeiteng** [comment]
I assume it runs on Linux.
